frontend/src/exploration/previews/layoutsData.ts

```diff
@@ -0,0 +1,78 @@
+import TabDefinition from '../../routing/TabDefinition';
+import LayoutDefinition from './LayoutDefinition';
+
+type LayoutData = {
+  label: string;
+  filename: string;
+  description: string;
+  weight: number;
+};
+
+export const layoutsData: {
+  [key: string]: LayoutData;
+} = {
```

Comment:
The following ensures that all keys of exploration weights are used
```suggestion
export const layoutsData: Record<keyof ExplorationWeight, LayoutData>
```